src/interactiveHandler.py

```diff
@@ -43,7 +43,7 @@ def getTransactionObject(self, command : str, networkHandler):
         tokens = self.inParser.lexer(command)
         if self.dummy:
             transactObj = self.getDummyTransactionObject(command, networkHandler)
-        elif tokens[self.serviceIdx] == "HOUSEKEEPING" and tokens[self.subserviceIdx] in ["GET_HK", "GET_INSTANT_HK"]:
+        elif tokens[self.serviceIdx] == "HOUSEKEEPING" and tokens[self.subserviceIdx] in ["GET_HK", "GET_INSTANT_HK", "GET_LATEST_HK"]:
```

Comment:
Can this be added to getDummyTransactionObject as well? or in fact, it might be best to just make an array called 'HK_COMMANDS' and just put all three commands there so we don't have to retype things.